### PR TITLE
Updated packages and ember-cli-deploy

### PR DESCRIPTION
**Pull request by aureliosaraiva**
https://github.com/ember-learn/builds/issues/44

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,4 +0,0 @@
-accessKeyId=ACCESSKEYID
-secretAccessKey=SECRETACCESSKEY
-bucket=bucket
-region=us-east-1
```
Comment (acorncom) [inline .env.example]: Is there a reason we're deleting this file?

Comment (aureliosaraiva) [reply .env.example]: I removed it, because it was written `example`, I thought it was rubbish. Only that.

I can reverse this if you prefer

Comment (locks) [reply .env.example]: Please do :) `.something.example` files are meant to illustrate what a `.something` should contain, without having to commit secrets, and without having to rename files and such. This way you can `.gitignore` the `.env` file and never accidentally leak secrets to the git repository.

Comment (locks) [reply .env.example]: Poke @aureliosaraiva

**Comment by locks** [comment]
Thanks for contributing @aureliosaraiva. Rob submitted https://github.com/ember-learn/builds/pull/58 with some more important changes, so I went with that one instead.

**Pull request closed without merging**